test/functional/p2p_segwit.py

```diff
@@ -514,7 +519,8 @@ def test_v0_outputs_arent_spendable(self):
             # 'non-mandatory-script-verify-flag (Witness program was passed an
             # empty witness)' (otherwise).
             # TODO: support multiple acceptable reject reasons.
```

Comment:
Is this TODO still relevant now that it explicitly sets single-threaded validation?

Reply:
Good question. One could argue that by implementing this TODO it'd be possible to further increase test coverage by asserting for the two different reject reasons dynamically, depending on whether `g_parallel_script_checks` is on; so on single-core systems, the specific message would be checked (like in this PR), and on multi-core systems, the more generic error message `block-validation-failed` would be checked. At least that's my interpreration of the TODO's original intention.

(Not sure though if it's worth it to do that. Currently, as far as I know we don't even have an easy way of determining of whether bitcoind uses parallel script verification if we don't pass `-par` explicitely. Also it makes the test less deterministic, and since most of the systems nowadays are multi-core, the case with the specific reject reasons would rarely be tested.)